modules/channel-messenger/src/backend/messenger.ts

```diff
@@ -159,8 +159,8 @@ export class MessengerService {
         debugMessages('could not find a bot for page id =', pageId)
         continue
       }
-
-      for (const webhookEvent of messages) {
+      if(messages !=undefined){ //TODO If its not a message process accordingly
```

Comment:
I'd just return after the `messages` assignation if `!messages`. Also can you make sure the file is properly formatted? :)

Reply:
sorry I did it using the github online edtiro, will do!